news/2017-11-05-Official-Mapping-Contests:-osu!-Mapping-Olympiad.md

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Rename this PR to appropriately represent what this article aims to do: examples are `add mapping olympiad newspost` or something among those lines. most prs that aren't fixes will just aim to create files so the title should describe what the file does

Reply:
done, hope i fixed the correct thing as i'm not quite sure what you meant